.github/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
 
 ### Improvements
 
+* Decouple LightningQubit memory ownership from numpy and migrate it to LightningQubit managed state-vector class.
+  [(#601)](https://github.com/PennyLaneAI/pennylane-lightning/pull/601)
+
+* Expand support for Projector observables on LightningKokkos.
+  [(#601)](https://github.com/PennyLaneAI/pennylane-lightning/pull/601)
```

Comment:
Think we usually write it more like ... for the release notes.
```suggestion
* Decouple LightningQubit memory ownership from numpy and migrate it to Lightning-Qubit managed state-vector class.
  [(#601)](https://github.com/PennyLaneAI/pennylane-lightning/pull/601)

* Expand support for Projector observables on Lightning-Kokkos.
  [(#601)](https://github.com/PennyLaneAI/pennylane-lightning/pull/601)
```

Reply:
Agree. Thank you!